app/helpers/application_helper.rb

```diff
@@ -22,16 +22,17 @@ def present(object, klass = nil)
   end
 
   def client_title(client)
-    html = ""
-    if client.logo_url != ""
-      html << image_tag(client.logo_url, class: "logo")
-    else
-      html << content_tag(
-        :span,
-        "",
-        class: "color",
-        style: "background-color:#{client.name.pastel_color};")
-    end
+    html = ''
+    html << if client.logo_url != ''
+              image_tag(client.logo_url, class: 'logo')
+            else
+              content_tag(
+                :span,
+                '',
+                class: 'color',
+                style: "background-color:#{client.name.pastel_color};"
+              )
+            end
```

Comment:
My editor auto-changed these files to satisfy rubocop, I can revert if wanted but don't have a strong preference

Reply:
I prefer te old style as it is more explicit (also have a tiny preference for double quotes) but w/e.

Reply:
yes! single quote vs double quote debate in 1,2,3...

It should match the rubocop defaults though, do you have your own config that your editor is user or is it the latest from rubocop itself?

Reply:
ok, I just noticed that you added a config file to the repo, but only to disable the rule. I'd say let it enforce single quotes to save you some work, or enforce the double quotes rule en run rubocop on the entire project folder.

Reply:
The default cop is single quotes nowadays, when Hound started out it had it's own custom configuration that preferred double quotes.

Since double quotes are already mostly enforced throughout the project I'll add a config to make that the default.